### PR TITLE
WIP: mesoshapes locality 1

### PR DESCRIPTION
**Pull request by nvkelso**
The easy peasy ones

### CODE REVIEW

**Comment by stepps00** [comment]
@nvkelso - Noting that this PR does not include deprecated `locality` features. This is likely correct, as Who's On First currently has more `locality` features than what's being imported. 

However, `locality` features will be deprecated in future meso locality imports (ex: [this branch](https://github.com/whosonfirst-data/whosonfirst-data/compare/stepps00/mesoshapes-locality-2)).

**Comment by nvkelso** [comment]
Re: https://github.com/whosonfirst-data/whosonfirst-data/pull/537#issuecomment-264296719, noting here that the no-dprecated `locality` features is deliberate (and correct).

File: data/101/749/143/101749143.geojson [hunk 1]
```diff
@@ -77,8 +88,8 @@
     "wof:lang":[
         "eng"
     ],
-    "wof:lastmodified":1467185322,
-    "wof:name":"\u03a0\u03ac\u03c6\u03bf\u03c2",
```
Comment (nvkelso) [inline data/101/749/143/101749143.geojson]: This name _was_ stored correctly.

File: data/101/749/145/101749145.geojson [hunk 2]
```diff
@@ -246,8 +257,8 @@
     "wof:lang":[
         "eng"
     ],
-    "wof:lastmodified":1467244766,
-    "wof:name":"\u039b\u03ac\u03c1\u03bd\u03b1\u03ba\u03b1",
+    "wof:lastmodified":1479929603,
+    "wof:name":"Larnaka",
```
Comment (nvkelso) [inline data/101/749/145/101749145.geojson]: This name _was_ stored correctly.

File: data/856/686/81/85668681.geojson [hunk 3]
```diff
@@ -58,8 +68,8 @@
     "wof:lang":[
         "eng"
     ],
-    "wof:lastmodified":1466633297,
-    "wof:name":"Nax\u00e7\u0131van",
+    "wof:lastmodified":1479926733,
+    "wof:name":"NAKHCHIVAN",
```
Comment (nvkelso) [inline data/856/686/81/85668681.geojson]: Should be Titlecase.

File: data/856/686/81/85668681.geojson [hunk 4]
```diff
@@ -58,8 +68,8 @@
     "wof:lang":[
         "eng"
     ],
-    "wof:lastmodified":1466633297,
-    "wof:name":"Nax\u00e7\u0131van",
```
Comment (nvkelso) [inline data/856/686/81/85668681.geojson]: Needs to be preserved as a name variant.

**Comment by thisisaaronland** [comment]
@nvkelso - yes,  the exportify code should update geom:area_square_m automatically.

**Comment by nvkelso** [comment]
@stepps00 can you ensure you have the latest exportify code installed, please?

**Comment by stepps00** [comment]
Closing and replacing.

**Comment by nvkelso** [comment]
Replacing with what?

**Comment by stepps00** [comment]
This PR and #544 will be replaced with a future PR. The recent locality import, coupled with merge conflicts, make these mesoshapes locality PRs outdated. Its best to close and redo this work to catch more existing records.

**Comment by stepps00** [comment]
The recent import of new records for https://github.com/whosonfirst-data/whosonfirst-data/pull/824 caused this PR to be out-of-date. 

A new PR to replace this PR will be linked once started.

**Pull request closed without merging**